src/freenet/node/NodeIPDetector.java

```diff
@@ -63,7 +65,7 @@ public void shouldUpdate(){
 	/** Explicit forced IP address in string form because we want to keep it even if it's invalid and therefore unused */
 	String overrideIPAddressString;
 	/** config: allow bindTo localhost? */
-	boolean allowBindToLocalhost;
+	Boolean allowBindToLocalhost;
```

Comment:
Is this actually necessary? The compiler should take care of all the boxing/unboxing…

Reply:
No it isn’t — thank you!